ml-algorithms/src/main/java/org/opensearch/ml/engine/utils/ZipUtils.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.engine.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.util.Enumeration;
+
+import org.apache.commons.compress.archivers.zip.ZipArchiveEntry;
+import org.apache.commons.compress.archivers.zip.ZipFile;
+import lombok.extern.log4j.Log4j2;
+
+/**
+ * A util class contains zip file related operations.
+ */
+@Log4j2
+public class ZipUtils {
+
+    /**
+     * Uncompressed a zip file.
+     * @param zipFile zip file to be uncompressed
+     * @param dest the destination path of this uncompress
+     */
+    public static void unzip(File zipFile, Path dest) {
+        try {
+            ZipFile file = new ZipFile(zipFile);
+            Enumeration<ZipArchiveEntry> en = file.getEntries();
+            ZipArchiveEntry ze;
+            while (en.hasMoreElements()) {
+                ze = en.nextElement();
+                String name = ze.getName();
+                Path f = dest.resolve(name).toAbsolutePath();
```

Comment:
minor: generally we don't use single char as variable name except for some special case like `i` in for loop.  How about change to more readable name?

Reply:
Corrected